### PR TITLE
 Intro-Python-I

### PR DESCRIPTION
**Pull request by vebradev**

### CODE REVIEW

File: src/01_bignum.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Print out 2 to the 65536 power
```
Comment (openwell) [inline src/01_bignum.py:1]: there is also a method approach... but yours is better.
method approach print(pow(int(2), int(65536)))

File: src/05_lists.py [hunk 2]
```diff
@@ -7,23 +7,23 @@
...
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 99, 10]
-# YOUR CODE HERE
+x.insert(5, 99)
 print(x)
 
 # Print the length of list x
-# YOUR CODE HERE
+print(len(x))
```
Comment (openwell) [inline src/05_lists.py:27]: Print all the values in x multiplied by 1000 not 2

File: src/08_comprehensions.py [hunk 3]
```diff
@@ -10,14 +10,14 @@
```
Comment (openwell) [inline src/08_comprehensions.py:10]: y = [n for n in range(1, 6)]
could also work

File: src/10_functions.py [hunk 4]
```diff
@@ -1,12 +1,20 @@
...
+
+
+print(is_even(2))  # True
+print(is_even(3))  # False
 
 # Read a number from the keyboard
 num = input("Enter a number: ")
 num = int(num)
 
 # Print out "Even!" if the number is even. Otherwise print "Odd"
```
Comment (openwell) [inline src/10_functions.py:19]: cleaver. good job

Comment (openwell) [reply src/10_functions.py:19]: You showed a mastery of git workflow by committing every step of the way.
You were able to complete the MVP.
Great work so far. Nothing to complain about.

File: src/01_bignum.py (hunk 1 above)
Comment (vebradev) [reply src/01_bignum.py:1]: It seems there's a method for everything in Python!

File: src/08_comprehensions.py (hunk 3 above)
Comment (vebradev) [reply src/08_comprehensions.py:10]: I like this one better. Will update the code.

**Comment by vebradev** [comment]
@openwell, pushed remaining exercises and added few changes after reading your comments. Thanks for your input!

**Review by openwell — APPROVED** [review]
Great work once again. 
All assignments completed.

**Pull request merged**